### PR TITLE
cleanup setup file

### PR DESCRIPTION
**Pull request by clevelam**
While updating the py-opppy spack environment for opppy-0_1_7 it was pointed out that the setup.py file should contain the dependencies for the opppy project. This includes the dependencies and updates the url in the setupfile.

### CODE REVIEW

**Comment by clevelam** [comment]
@tylerjereddy will you look this over when you have time?

File: setup.py [hunk 1]
```diff
@@ -31,10 +31,12 @@
 
 setup(name='opppy',
       version=__version__,
+      python_requires='>=3.0',
```
[the remark this answers is not in the document]
Comment (clevelam) [reply setup.py]: Thanks for the helpful feedback. I did bump it up to 3.6 (I need to ensure support at least back this far for other related projects). That numpy policy is very handy and I will keep it in mind going forward.

File: setup.py [hunk 2]
```diff
@@ -31,10 +31,12 @@
...
       description='Output Parse-Pickle-Plot Python (OPPPY) library',
       author='Mathew Cleveland',
       author_email='cleveland@lanl.gov',
-      url='https://gitlab.lanl.gov/chaos/opppy',
+      url='https://github.com/lanl/OPPPY',
+      install_requires=['numpy', 'scipy', 'matplotlib'],
```
[the remark this answers is not in the document]
Comment (clevelam) [reply setup.py:39]: Another helpful point. Is pyproject.toml intended to replace the old setup.py format? I notice that you have both in your forked repo.

[the remark this answers is not in the document]
Comment (tylerjereddy) [reply setup.py:39]: In some cases it does--certainly it is a staple of the modern wheel-building ecosystem.

[the remark this answers is not in the document]
Comment (clevelam) [reply setup.py:39]: Thanks. I'm opening a ticket with this info.

**Pull request merged**